test/gerkin/tests/features/data_read_value.feature

```diff
@@ -0,0 +1,16 @@
+Feature: API endpoint allowing users to search for one record's field value in the Digital Registries database.
+  Request endpoint: GET /data/{registryname}/{versionnumber}/{uuid}/read-value/{field}.{ext}
+
+  Background:
+    Given The user wants to search for the first name of the user with uuid "123" in the Digital Registries database
```

Comment:
`123` is not a valid uuid, please change it.
You can specify first name as FirstName which is a valid field name for this BB